src/index.html

```diff
@@ -1,13 +1,46 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <section class="item" data-qa="card">
+      <img
+        class="item__image"
+        src="./images/imac.jpeg"
+        alt="Моноблок"
+      >
+      <h1 class="item__name">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </h1>
+      <p class="item__code">Код товара: 195434</p>
+      <div class="item__rating">
+        <div class="item__stars">
```

Comment:
```suggestion
        <div class="item__stars item__stars--4">
```
модификатор нужно тут делать